Gems/ROS2/Code/ros2_header_files.cmake

```diff
@@ -19,8 +19,8 @@ set(FILES
         Include/ROS2/Sensor/ROS2SensorComponent.h
         Include/ROS2/Sensor/SensorConfiguration.h
         Include/ROS2/Spawner/SpawnerBus.h
+        Include/ROS2/Utilities/Controllers/PidConfiguration.h
         Include/ROS2/Utilities/ROS2Conversions.h
         Include/ROS2/Utilities/ROS2Names.h
-        Include/ROS2/VehicleDynamics/DriveModels/PidConfiguration.h
```

Comment:
This change possibly breaks apple orchard demo with development. However, since we decided to support exact commit hashes and upgrade gradually, this is ok.